user-guide/Advanced_Functionality/Swarming/EnableSwarming.md

```diff
@@ -0,0 +1,102 @@
+---
+uid: EnableSwarming
+---
+
+# Enabling the Swarming feature
+
+> [!IMPORTANT]
+> We highly recommend creating a backup before enabling the Swarming feature.
+
+## Prerequisites
+
+- DataMiner 10.5.1 or higher.
+- [STaaS](xref:STaaS) or a [dedicated clustered storage](xref:Configuring_dedicated_clustered_storage) setup
+- No [Failover Agents](xref:About_DMA_Failover) are present in the cluster.
+- No [offload database](xref:Offload_database) is configured.
+- Enhanced services in the cluster (if any) use only [compatible connectors](xref:SwarmingPrepare).
+- Scripts (Automation/GQI) and QActions in connectors have all been [made compatible](xref:SwarmingPrepare), i.e. they do not use obsolete or incompatible SLNet calls/properties that handle Alarm IDs as "DmaID/AlarmID".
+- For the user account that will be used to enable Swarming, the [Admin Tools](xref:DataMiner_user_permissions#modules--system-configuration--tools--admin-tools) user permission is enabled.
+
+## Running a prerequisites check
+
+To verify whether your system meets these prerequisites before you enable Swarming, you can send a *SwarmingPrerequisitesCheckRequest* message to SLNet using the [SLNetClientTest tool](xref:SLNetClientTest_tool). The prerequisite check is also executed when you enable Swarming, ensuring that all prerequisites are met before Swarming is enabled.
+
+> [!NOTE]
+> The prerequisites check can take some time, as checking the usage of legacy alarm IDs in scripts can take a while.
+
+To run a prerequisites check using SLNetClientTest tool:
+
+1. [Connect to the DMA using the SLNetClientTest tool](xref:Connecting_to_a_DMA_with_the_SLNetClientTest_tool).
+
+   > [!CAUTION]
+   > Always be extremely careful when using this tool, as it can have far-reaching consequences on the functionality of your DataMiner System.
+
+1. Go to the *Build Message* tab of the main window of the SLNetCLientTest tool.
+
+1. In the *Message Type* drop-down list, select the message *Skyline.DataMiner.Net.Swarming.SwarmingPrerequisitesCheckRequest*.
```

Comment:
should we guide the user in skipping the alarmid check if they want to check the other things first or later on repeat?

Reply:
I've now added additional info in line 40 for this. Could you check if this is OK?

Reply:
ok for me